lib/Mail.php

```diff
@@ -238,4 +251,16 @@ private function createMailBody($filename, $link, $passwordLink, $cloudName, $di
 
 		return [$htmlMail, $plainTextMail];
 	}
+
+	/**
+	 * get default_language if defined in config.php
+	 */
+	private function getDefaultLanguage() {
+		$useDefaultLanguage = null;
+		$defaultLang = $this->config->getSystemValue('default_language', false);
+		if ($defaultLang !== false) {
+			$useDefaultLanguage = \OC::$server->getL10N('lib', $defaultLang);
+		}
+		return $useDefaultLanguage;
```

Comment:
```suggestion
		return $defaultLanguage;
```
This is a string in the config like `en_GB` so the var name could be changed so that it does not sound like a boolean.
And add PHPdoc for the return type `?string`